spec/classes/sensu_package_spec.rb

```diff
@@ -40,7 +40,6 @@
 
         it { should contain_package('sensu-plugin').with(
           :ensure   => 'installed',
-          :provider => 'gem'
```

Comment:
I thought about that, but I think specifying the provider here is superfluous since the context of the test, "setting version with / without platform suffix" doesn't seem to relate behaviorally to the provider.  I would have left it the provider wasn't fully covered in the embedded_ruby context.  Happy to add them back if you prefer.